modules/src/core/ics02_client/context.rs

```diff
@@ -124,4 +129,14 @@ pub trait ClientKeeper {
     /// Increases the counter which keeps track of how many clients have been created.
     /// Should never fail.
     fn increase_client_counter(&mut self);
+
+    /// Called upon successful client update.
+    /// Implementations are expected to use this to record the current (host) time as the time at
+    /// which this update (or header) was processed.
+    fn store_processed_time(&mut self, client_id: ClientId, height: Height) -> Result<(), Error>;
+
+    /// Called upon successful client update.
+    /// Implementations are expected to use this to record the current (host) height as the height
+    /// at which this update (or header) was processed.
+    fn store_processed_height(&mut self, client_id: ClientId, height: Height) -> Result<(), Error>;
```

Comment:
The terminology is a bit unclear. Would it be more accurate if we call these methods `store_update_*` instead of `store_processed_*`?

```suggestion
    fn store_update_time(&mut self, client_id: ClientId, height: Height) -> Result<(), Error>;

    /// Called upon successful client update.
    /// Implementations are expected to use this to record the current (host) height as the height
    /// at which this update (or header) was processed.
    fn store_update_height(&mut self, client_id: ClientId, height: Height) -> Result<(), Error>;
```